### PR TITLE
Remove file ext validation

### PR DESCRIPTION
**Pull request by Nimond**
# Checklist

- [x] Linters and tests have passed
- [x] I've written a changelog for PR change
- [x] I'll make a release when PR is approved

### CODE REVIEW

**Review by Opti213 — APPROVED** [review]

File: botx/models/files.py [hunk 1]
```diff
@@ -229,36 +264,20 @@ def media_type(self) -> str:
...
-        Raises:
-            ValueError: when mimetype is unsupported.
+            file extension or none if mimetype not found.
         """
         for ext, m_type in EXTENSIONS_TO_MIMETYPES.items():
             if m_type == mimetype:
                 return ext
 
-        raise ValueError("`{0}` is unsupported mimetype.".format(mimetype))
+        return None
```
Comment (Opti213) [inline botx/models/files.py:280]: Maybe `""`?

Comment (lxmnk) [reply botx/models/files.py:280]: It's not a mimetype.

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "botx"
-version = "0.23.2"
+version = "0.23.3"
```
Comment (lxmnk) [inline pyproject.toml:3]: `0.24`?

File: botx/models/files.py (hunk 1 above)
Comment (Nimond) [reply botx/models/files.py:280]: But `""` is not a valid extension, meanwhile `None` says that ext not found

File: pyproject.toml (hunk 2 above)
Comment (Nimond) [reply pyproject.toml]: hmm, ok

**Review by lxmnk — APPROVED** [review]

**Review by rrrrs09 — APPROVED** [review]

**Pull request merged**